codec-http/src/test/java/io/netty/handler/codec/http/HttpClientCodecTest.java

```diff
@@ -186,12 +186,12 @@ protected void channelRead0(ChannelHandlerContext ctx, FullHttpResponse msg) {
                 }
             });
 
-            Channel serverChannel = sb.bind(new InetSocketAddress(0)).sync().channel();
+            Channel serverChannel = sb.bind(new InetSocketAddress(0)).sync().getNow();
             int port = ((InetSocketAddress) serverChannel.localAddress()).getPort();
 
-            ChannelFuture ccf = cb.connect(new InetSocketAddress(NetUtil.LOCALHOST, port));
+            Future<Channel> ccf = cb.connect(new InetSocketAddress(NetUtil.LOCALHOST, port));
             assertTrue(ccf.awaitUninterruptibly().isSuccess());
-            Channel clientChannel = ccf.channel();
+            Channel clientChannel = ccf.getNow();
```

Comment:
We can just do

```java
Channel clientChannel = cb.connect(new InetSocketAddress(NetUtil.LOCALHOST, port)).get();
```

An let any failure propagate from `get()` - we'd get more info than the `assertTrue` failing.